tox.ini

```diff
@@ -649,7 +649,7 @@ commands =
     ; Running `py.test` as an executable suffers from an import error
     ; when loading tests in scenarios. In particular, django fails to
     ; load the settings from the test module.
-    {py3.6,py3.7,py3.8,py3.9,py3.10,py3.11,py3.12}: python -m pytest -rsx -s --durations=5 -vvv {env:TESTPATH} {posargs}
+    {py3.6,py3.7,py3.8,py3.9,py3.10,py3.11,py3.12}: python -m pytest -rsfx -s --durations=5 -vvv {env:TESTPATH} {posargs}
```

Comment:
what does the changed argument do?

Reply:
Prints in the summary a lift of all tests failing: https://docs.pytest.org/en/7.1.x/how-to/output.html#producing-a-detailed-summary-report

Reply:
Cool!